cheat.php

```diff
@@ -778,3 +789,4 @@ function Msg( $Message, $EOL = PHP_EOL, $printf = [] )
 		echo $Message;
 	}
 }
+# vim: set tabstop=8 softtabstop=8 shiftwidth=8 noexpandtab :
```

Comment:
undo this

Reply:
Are you sure? It's just a comment to help VIM users like myself who don't use tabulations. When I hit TAB, the VIM editor converts them to 4 spaces instead. This line I added just prevents that...

Reply:
If we stored configs for each unique editor, it would be a mess.

Reply:
Fair enough!